.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+AllCops:
+  DisabledByDefault: true
+
+Security:
+  Enabled: true
+
+Bundler/DuplicatedGem:
+  Enabled: true
+Bundler/InsecureProtocolSource:
+  Enabled: true
+Layout/IndentationStyle:
+  Enabled: true
+Layout/SpaceInsideRangeLiteral:
+  Enabled: true
+Lint:
+  Enabled: true
+Lint/RaiseException:
```

Comment:
redundant with `Lint`?

Reply:
No, otherwise `rubocop` asks if you want this / asks for you to add explicitly. I'm not sure why tbh.